tests/Image/ImageServiceTest.php

```diff
@@ -12,19 +12,30 @@
 
 final class ImageServiceTest extends ApiTestCase
 {
+    private $id = 12;
+
     public function testGetImageLink()
     {
-        $this->assertSame('http://wizaplace.loc/api/v1/image/1', (string) $this->buildImageService()->getImageLink(1));
+        $this->assertSame(
```

Comment:
pareil pour les autres tests plus bas
```suggestion
        static::assertSame(
```

Reply:
Allright